R/execution.R

```diff
@@ -133,6 +40,9 @@ executeDqChecks <- function(connectionDetails,
                             cdmDatabaseSchema,
                             resultsDatabaseSchema,
                             cdmSourceName,
+                            cohort = FALSE,
```

Comment:
Can you add these to the meta tagging of the function?

Reply:
Also, do we need the cohort parameter? Could the presence of cohortDefinitionIds indicate that we want to run this for a cohort or cohorts?